app/download/page.tsx

```diff
@@ -43,20 +47,33 @@ export default function Download() {
               employeeId
             </th>
             <th scope="col" className="px-6 py-4 font-medium text-gray-900">
-              download
+              phone
+            </th>
+            <th scope="col" className="px-6 py-4 font-medium text-gray-900">
+              mail
             </th>
             <th scope="col" className="px-6 py-4 font-medium text-gray-900">
-              なんか入れられる
+              ?
             </th>
           </tr>
         </thead>
         <tbody className="divide-y divide-gray-100 border-t border-gray-100">
           <tr className="hover:bg-gray-50">
+            <th className="px-6 py-4">
+              <div className="flex h-5 items-center">
+                <input
+                  type="checkbox"
+                  id="example12"
```

Comment:
Do not name unrelated id.
Would you reset id value?
Or, delete id.

Reply:
Deleted id.